ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -882,6 +892,19 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         }
     }
 
+    /// The custom view on the leading edge of the `customView` UIView. Does not display if layoutType is `.oneLine`.
+    @objc open var cellLeadingView: UIView? {
+        didSet {
+            if layoutType == .oneLine {
```

Comment:
having this customizable cellleadinView is awesome! but i think we should also havea default circle drawn if it is part of fluent library now?

Reply:
I was thinking this as well but wanted to leave options open just in case! It's a quick fix, so I'll go ahead and make that change.

Reply:
follow example of badges how we use CALayer to draw tiny dots.  layers are more efficient than imageview sometimes.

Reply:
Wait sorry, should I have a fallback default circle, or circle only and turn this into a Bool? (etc. showCellLeadingView)

Reply:
Resolved offline: Passing in a bool and setting circle internally